components/multiselect/multiselect.jsx

```diff
@@ -264,7 +264,7 @@ export default class MultiSelect extends React.Component {
                 if (this.state.page > 0) {
                     previousButton = (
                         <button
-                            className='btn btn-default filter-control filter-control__prev'
+                            className='btn btn-link filter-control filter-control__prev'
```

Comment:
I think this fix is for [MM-13811](https://mattermost.atlassian.net/browse/MM-13811) which has PR submitted - https://github.com/mattermost/mattermost-webapp/pull/2280/files